### PR TITLE
Add geodesic package for geodesic calculation

### PR DESCRIPTION
**Pull request by Yaqiang**
In EquidistantAzimuthalProjection, project and projectInverse functions results are not corrected with elliptical OBLIQUE or EQUATOR setting #84. Geodesic calculation is needed in this condition, so geodesic package was added in proj4j with the code from GeographicLib-Java project (https://github.com/geographiclib/geographiclib-java). Also I added testEquidistantAzimuthal function for testing, and it passed.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -90,7 +90,7 @@
                             <goal>jar</goal>
                         </goals>
                         <configuration>
-                            <failOnError>true</failOnError>
+                            <failOnError>false</failOnError>
```
Comment (pomadchin) [inline pom.xml:93]: What is the reason for this change?

**Review by pomadchin — APPROVED** [review]
Hey! That is a great PR! I left he question is only with regards to the pom.xml changes, rather then that I see nothing that prevents it from being merged.

**Comment by pomadchin** [comment]
Could I ask you to also update the [CHANGELOG.md](https://github.com/locationtech/proj4j/blob/master/CHANGELOG.md)? 

Into the Unreleased Fix section, the sape here is "* the PR name `issue #`"

File: pom.xml (hunk 1 above)
Comment (Yaqiang) [reply pom.xml:93]: Sorry I forgot the reason for this change (^-^). Keep `true` should be ok. Could you solve this on your side or I have to close this PR and re-commit a new one?

By the way, jgitver extension does not work correctly on my side, which always make 1.1.2-SNAPSHOT version.

Comment (pomadchin) [reply pom.xml:93]: Let's keep it as is, would be extra cool if you could reflect that in the CHANGEFLOG as a separate bullet point. 

About jgitver: that's weird, probably you're testing it off a fork and there are no tags fetched; that's what the master branch prints me:

```bash
(master)]$ mvn package
[INFO] Using jgitver configuration file: /Users/.../proj4j/.mvn/jgitver.config.xml
[INFO] Using jgitver-maven-plugin [1.8.0] (sha1: 3f3523c5f7540865e3ad66fc9ef1c9daa52a6e6b)
[INFO]     version '1.1.6-SNAPSHOT' computed in 1340 ms
[INFO]
[INFO] Scanning for projects...
[INFO] jgitver-maven-plugin is about to change project(s) version(s)
[INFO]     org.locationtech.proj4j::proj4j::1.1.6-SNAPSHOT -> 1.1.6-SNAPSHOT
[INFO]
[INFO] -------------------< org.locationtech.proj4j:proj4j >-------------------
[INFO] Building Proj4J 1.1.6-SNAPSHOT
```

**Comment by Yaqiang** [comment]
> Could I ask you to also update the [CHANGELOG.md](https://github.com/locationtech/proj4j/blob/master/CHANGELOG.md)?
> 
> Into the Unreleased Fix section, the sape here is "* the PR name `issue #`"

I will update the change log file. How about this PR? Should I close it firstly?

I have anothor waiting PR for Robinson projection bug fix (https://github.com/Yaqiang/proj4j/commit/bece41a21b93be947f7a17122b6483b6033934ce). Maybe I can close this one and merge the two PR, then re-commit it? How about your suggestion?

File: pom.xml (hunk 1 above)
Comment (Yaqiang) [reply pom.xml]: I have revised it as `true` now.

**Comment by pomadchin** [comment]
@Yaqiang 💯 could I ask you to commit --amend to trigger ECA bot one more time? I think it's sime kind of a lag.

**Comment by Yaqiang** [comment]
> @Yaqiang 💯 could I ask you to `commit --amend` extra time to trigger ECA bot one more time? I think it's sime kind of a lag.

Sorry for my lacking knowledge of git and github. I don't know how to do it detailed.
